governance/votes/Executive vote - November 19, 2021.md

```diff
@@ -0,0 +1,168 @@
+- A change of covenants for P1-DROP (Peoples Company Series 1).
+- BAL-A, COMP-A, AAVE-A, LRC-A, BAT-A, ZRX-A, UNIV2LINKETH-A will be offboarded
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- WBTC-B will be onboarded as a new vault type.
+
+More details on these changes and additions can be found in the _Proposal Details_ section below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### WBTC-A Stability Fee will be increased from 2.5% to 4.0%
+
+As per the outcome of [this poll](https://vote.makerdao.com/polling/QmRUgsvi), the following [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) will be **increased** if this executive proposal passes:
+
+| VAULT TYPE | CURRENT SF | PROPOSED SF | SF DIFFERENCE |
+|--|--|--|--|
+| WBTC-A | 2.5% | 4% | +1.5% |
+
+### Increases to Local Liquidation Limits (`ilk.hole`) for ETH-A, ETH-B, ETH-C, WBTC-A, WSTETH-A
+
+As per the outcome of [this poll](https://vote.makerdao.com/polling/QmQN6FX8), the following [Local Liquidation Limits](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) will be **increased** if this executive proposal passes:
+
+| VAULT TYPE | CURRENT LLL | PROPOSED LLL | LLL DIFFERENCE |
+|--|--|--|--|
+| ETH-A | 40 million DAI | 65 million DAI | +25 million DAI |
+| ETH-B | 25 million DAI | 30 million DAI | +5 million DAI |
+| ETH-C | 30 million DAI | 35 million DAI | +5 million DAI |
+
+### A change of Covenants for P1-DROP (Peoples Company Series 1)
+
+As per [this poll], the following covenant changes to P1-DROP (Peoples Company Series 1) will occur if this executive proposal passes:
+
+- Increase maximum acquisition price from $1.5 million to $2.85 million.
+- Include Arkansas as a possible state.
+
+### Offboarding Vaults
+
+The listed vaults will be offboarded by implementing the following changes if this executive proposal passes:
```

Comment:
I would say "following" instead of "listed," makes it sound a bit more natural

Reply:
Hah. No idea why I wrote "listed" :man_shrugging: